src/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
And where exactly is the element with id `root` ? I can't find it in the `public/index.html` 😳

Reply:
Added root id div back into html file